addons/mail/static/src/owl/store/messaging_actions.js

```diff
@@ -979,6 +1007,20 @@ const actions = {
         dispatch('closeChatWindow', oldChatWindowLocalId);
         dispatch('focusChatWindow', newChatWindowLocalId);
     },
+    /**
+     * @param {Object} param0
+     * @param {function} param0.dispatch
+     * @param {Object} param0.state
+     * @param {string} composerLocalId
+     */
+    resetComposer({ dispatch, state }, composerLocalId) {
```

Comment:
Shouldn't posted attachments also be reset? It might be done elsewhere (?) but I'm surprised to not see it here.

Reply:
Done